.github/workflows/integration.yml

```diff
@@ -92,6 +93,27 @@ jobs:
             ginkgo --focus="E2E TEST:executable|E2E TEST:blob" ./...
           fi
 
+      - name: Build docker image
+        run: docker build . --file Dockerfile --tag $IMAGE_NAME
+
+      - name: Test docker image
+        env:  
+          GITHUB_AUTH_TOKEN: ${{ secrets.GH_AUTH_TOKEN }}
+        run: |
+          if [[ ! -v GITHUB_AUTH_TOKEN ]]; then
+              echo "GITHUB_AUTH_TOKEN is not set, skip test"
+              exit 1
+          elif [[ -z "$GITHUB_AUTH_TOKEN" ]]; then
+              echo "GITHUB_AUTH_TOKEN is set to the empty string, skip test"
+              exit 1
+          else
+            go get github.com/onsi/ginkgo/ginkgo@v1.14.2
+            go mod download
+           	mkdir -p bin
```

Comment:
maybe we should put results in output directory instead of bin. we do this in one more place.
also indent seems off by 2 spaces.

Reply:
Agreed, we need to put the results in the output directory. I will fix that part of another PR for the entire repo.